volontulo/urls.py

```diff
@@ -23,27 +23,37 @@
     # me/settings
 
     # offers' namesapce:
-    url(r'^offers', views.list_offers, name='list_offers'),
+    url(r'^offers$', views.list_offers, name='list_offers'),
     url(
-        r'^offers/activate/(?P<offer_id>[0-9]+)$',
-        views.activate_offer,
-        name='activate_offer'
-    ),
-    url(
-        r'^offers/show/(?P<offer_id>[0-9]+)$',
+        r'^offers/(?P<slug>[\w-]+)/(?P<offer_id>[0-9]+)$',
         views.show_offer,
         name='show_offer'
     ),
     url(
-        r'^offers/form/(?P<organization_id>[0-9]+)$',
+        r'^offers/create$',
         views.offer_form,
         name='offer_form'
     ),
+    # it should be moved to offers/<slug>/<id>/edit:
     url(
-        r'^offers/apply/(?P<offer_id>[0-9]+)$',
+        r'^offers/activate/(?P<offer_id>[0-9]+)$',
```

Comment:
But instead of applu we use verb join. They the same thing in our functionality.